AdminWebsite/AdminWebsite.UnitTests/Controllers/WorkHoursControllerTests.cs

```diff
@@ -0,0 +1,49 @@
+using AdminWebsite.Controllers;
+using BookingsApi.Client;
+using BookingsApi.Contract.Requests;
+using BookingsApi.Contract.Responses;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace AdminWebsite.UnitTests.Controllers
+{
+    public class WorkHoursControllerTests
+    {
+        private Mock<IBookingsApiClient> _bookingsApiClientMock;
+
+        private List<string> _failedUsernames;
+
+        private WorkHoursController _controller;
+
+        [SetUp]
+        public void Setup()
+        {
+            _failedUsernames = new List<string> { "failedusername@test.com" };
+
+            _bookingsApiClientMock = new Mock<IBookingsApiClient>();
+            _bookingsApiClientMock.Setup(x => x.SaveWorkHoursAsync(It.IsAny<List<UploadWorkHoursRequest>>()))
```

Comment:
could we maybe move this failed usernames setup into the test instead, so that it won't return failed usernames if we add more tests?